core/src/main/java/com/web3auth/core/Web3Auth.kt

```diff
@@ -102,23 +130,124 @@ class Web3Auth(web3AuthOptions: Web3AuthOptions) {
             logoutCompletableFuture.complete(null)
         }
 
+        web3AuthResponse.sessionId?.let { KeyStoreManagerUtils.encryptData(KeyStoreManagerUtils.SESSION_ID, it) }
```

Comment:
what's the key it's being stored under?
it should be verifier + " | " + verifierId

Reply:
Was using key "sessionId". But updated now.